sdv/constraints/tabular.py

```diff
@@ -388,92 +403,91 @@ def reverse_transform(self, table_data):
+            new_values = pd.DataFrame(diff + low, columns=self._high)
+            table_data[self._high] = new_values.astype(dict(zip(self._high, self._dtype)))
         elif self._drop == 'low':
-            table_data[self._low] = (high - diff).astype(self._dtype)
+            new_values = pd.DataFrame(high - diff, columns=self._low)
+            table_data[self._low] = new_values.astype(dict(zip(self._low, self._dtype)))
         else:
             invalid = ~self.is_valid(table_data)
             if not self._high_is_scalar and not self._low_is_scalar:
-                new_values = low.loc[invalid] + diff.loc[invalid]
+                new_values = low[invalid] + diff[invalid]
             elif self._high_is_scalar:
-                new_values = high - diff.loc[invalid]
+                new_values = high - diff[invalid]
             else:
-                new_values = low + diff.loc[invalid]
+                new_values = low + diff[invalid]
 
-            table_data[self._column_to_reconstruct].loc[invalid] = new_values.astype(self._dtype)
+            for i, column in enumerate(self._columns_to_reconstruct):
+                table_data.at[invalid, column] = new_values[:, i].astype(self._dtype[i])
 
-        table_data = table_data.drop(self._diff_column, axis=1)
+        table_data = table_data.drop(self._diff_columns, axis=1)
 
         return table_data
 
 
 class Positive(GreaterThan):
-    """Ensure that the ``high`` column is always positive.
+    """Ensure that the given column is always positive.
 
-    The transformation strategy works by creating a column with the
-    difference between ``high`` and 0 value and then computing back the ``high``
-    value by adding the difference to 0 when reversing the transformation.
+    The transformation strategy works by creating columns with the
+    difference between given columns and zero then computing back the
+    necessary columns using the difference.
 
     Args:
-        high (str or int):
-            Either the name of the column that contains the high value,
-            or a scalar that is the high value.
+        columns (str or list[str]):
+            The name of the column(s) that are constrained to be positive.
         strict (bool):
-            Whether the comparison of the values should be strict ``>=`` or
-            not ``>`` when comparing them. Currently, this is only respected
+            Whether the comparison of the values should be strict; disclude
+            zero ``>`` or include it ``>=``. Currently, this is only respected
             if ``reject_sampling`` or ``all`` handling strategies are used.
         handling_strategy (str):
             How this Constraint should be handled, which can be ``transform``
             or ``reject_sampling``. Defaults to ``transform``.
-        drop (str):
-            Which column to drop during transformation. Can be ``'high'``
-            or ``None``.
+        drop (bool):
```

Comment:
`drop` is a `bool` here but a `str` in the `GreaterThan` constraint. I see that below we're just passing the `drop` argument straight through to `super().__init__`. Do we need some kind of translation? e.g.
```
if drop:
  drop = 'high'
```

Same question for the `Negative` constraint below.

Reply:
We could add unit tests for `Positive` and `Negative` where `drop` is set to a non-None value.

Reply:
I also find it a little confusing that `drop` can be `bool` in some constraints, but `str` in others. We can discuss this as a larger topic though.

Reply:
good catch!